Rakefile

```diff
@@ -0,0 +1,12 @@
+# require 'data_mapper'
+# require './app/data_mapper_setup'
+
+# task :auto_upgrade do
+#   DataMapper.auto_upgrade!
+#   puts "Auto-upgrade complete (no data loss)"
+# end
+
+# task :auto_migrate do
+#   DataMapper.auto_migrate!
+#   puts "Auto-migrate complete (data could have been lost)"
+# end
```

Comment:
Try not to leave commented code lying around - people coming to your code later on (or you) won't know whether it's dead code to be deleted or something important